probes/hasDangerousWorkflowScriptInjection/patch/testdata/badIndentationMultipleInjections.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+  on: [pull_request]
+
+jobs:
+      build:
+    name: Build and test
```

Comment:
This isn't valid GitHub Action syntax. Remember that YAML requires consistent indentation. If `build` is a job name (the thing expected under `jobs`), then what is `name`? It's in a separate indentation level but not the same indentation as `jobs`, so it's not in the global scope either.

I'm 100% in favor of bad indentation to make sure the remediation keeps the existing style, but I don't think it needs to be tested on an invalid workflow (unless we want to check that it "fails gracefully").

Reply:
Hey Pedro! No, I just didn't know that this would fail. Given that workflows with bad indentation work, I was assuming it didn't care much about indentation.

So, how can I place the indentation in order to make it a bad indentation, but still run on GitHub? Make Indentation with arbitrary number of Tabs instead of always 1, but still keeping them consistent between themselves?

Reply:
I'm taking a look at https://learnxinyminutes.com/docs/yaml/ and I think I'm already able to follow on this. I'll let you know if I still need help